cmd/root.go

```diff
@@ -44,10 +43,10 @@ func init() {
 	rootCmd.PersistentFlags().IntVar(&credential.ServerPort, "port", 7091, "Seata Server Admin Port")
 	rootCmd.PersistentFlags().StringVar(&credential.Username, "username", "seata", "Username")
 	rootCmd.PersistentFlags().StringVar(&credential.Password, "password", "seata", "Password")
-	viper.BindPFlag("ip", rootCmd.PersistentFlags().Lookup("ip"))
-	viper.BindPFlag("port", rootCmd.PersistentFlags().Lookup("port"))
-	viper.BindPFlag("username", rootCmd.PersistentFlags().Lookup("username"))
-	viper.BindPFlag("password", rootCmd.PersistentFlags().Lookup("password"))
+	//viper.BindPFlag("ip", rootCmd.PersistentFlags().Lookup("ip"))
+	//viper.BindPFlag("port", rootCmd.PersistentFlags().Lookup("port"))
+	//viper.BindPFlag("username", rootCmd.PersistentFlags().Lookup("username"))
+	//viper.BindPFlag("password", rootCmd.PersistentFlags().Lookup("password"))
```

Comment:
Comment those lines will affect original usage. So don't comment.

Reply:
This comment is related to the startup of the main program, and the relevant logic has been corrected